src/module_wrap.cc

```diff
@@ -147,14 +147,14 @@ void ModuleWrap::New(const FunctionCallbackInfo<Value>& args) {
   // compile
   {
     ScriptOrigin origin(url,
-                        line_offset,                          // line offset
```

Comment:
Can you please revert all of the unrelated whitespace changes in this file and `src/node_contextify.cc`.

Reply:
options are:
1. revert whitespace changes and keep the more accurate comment ("is shareable cross origin") but violate 80-char limit
2. revert whitespace changes and return to the less accurate comment ("is cross origin"), staying within 80-char limit
3. revert whitespace changes on other lines, keep the more accurate comment, and have unaligned comments
4. keep the whitespace changes

which would you prefer?

Reply:
I'd say option 2 or 3. I don't think the updated comment is **that** much more helpful (but others might disagree). The unrelated lines should definitely be reverted to keep the diff size smaller and easier to review, and also for future `git blame`s.